### DISCUSSION TITLE
Support for responsiveness measurements

### DISCUSSION THREAD

**Discussion by AMichl** [esnet/iperf#1725] [Ideas]
I would like to propose an enhancement to allow responsiveness testing, i.e. providing RTT measurements along with UDP or TCP throughput scenarios.

Responsiveness testing is a vital component in both software and hardware development across multiple industries. As virtual reality devices continue to evolve, this form of testing has gained significant prominence, especially in the realm of wireless communication networks, where ensuring timely and efficient data transfer is crucial for an optimal user experience.

The iperf2 development team has already addressed this subject, and the latest release, version 2.2.0, includes new capabilities that facilitate the measurement of Round-Trip Time (RTT) even while generating significant data traffic in either the uplink or downlink direction. These features can be accessed using the '--bounceback' and '--working_load' command-line arguments.

**Comment by davidBar-On** [comment]
Submitter PR  #1783 with a suggested implementation of Bounceback tests in iperf3.
